029-IoTEdge/Student/Challenge-07.md

```diff
@@ -0,0 +1,116 @@
+# Challenge 7: Connect to Devices with Device Streams
+
+[< Previous Challenge](./Challenge-06.md) - **[Home](../README.md)**
+
+## Pre-requisites
+
+For this challenge you will need the following:
+
+- A `SSH` client installed. On Windows, this is available out-of-the-box in recent builds of Windows 10 and in the Windows Subsystem for Linux (both 1 and 2). You can also download a simple client if needed, such as [PuTTY](https://www.putty.org/). It should also be available out-of-the-box in most Linux distributions and MacOS.
```

Comment:
SSH in ssh client should be monospaced, it isn't really a variable name or command.... I would just call it an 'ssh client' or use uppercase SSH if you wish.

Same applies to the rest of this file... just use ssh in regular text, no backticks.